resources/js/Pages/Praktikum.vue

```diff
@@ -592,14 +592,14 @@
                 TA
               </span> 
               <span class="font-overpass-bold text-9xl " 
-                  :class="[{ 'visible': statusPraktikum === 2 },
-                          { 'hidden': statusPraktikum != 2 }]">
-                PRESENTASI
+                  :class="[{ 'visible': statusPraktikum === 2 || statusPraktikum === 123 },
```

Comment:
Reminder: status `123` is for `runmod` purposes, look up into previous periods `runmod` rundown!

Reply:
@Soedipo so you add this with purpose if runmod mode practicum then praktikan can access TA section? that's right?

Reply:
yes, but TA and TK section will be skipped by PJ when runmod, is it ok?

Reply:
okeyy, good

> yes, but TA and TK section will be skipped by PJ when runmod, is it ok?